packages/ui-popover/src/Popover/index.tsx

```diff
@@ -237,7 +237,9 @@ class Popover extends Component<PopoverProps> {
     if (
       this.props.offsetX !== prevProps.offsetX ||
       this.props.offsetY !== prevProps.offsetY ||
-      this.props.placement !== prevProps.placement
+      this.props.placement !== prevProps.placement ||
+      this.props.shouldAlignArrow !== prevProps.shouldAlignArrow ||
+      this.props.withArrow !== prevProps.withArrow
```

Comment:
You could write a test for this, for example like here: https://github.com/instructure/instructure-ui/blob/datetimeinput/packages/ui-date-time-input/src/DateTimeInput/__tests__/DateTimeInput.test.tsx#L442

Reply:
Unfortunately, our current testing lib does not allow for state checking, so the testing not really viable